data/src/main/java/org/apache/iceberg/data/TableMigrationUtil.java

```diff
@@ -215,11 +215,11 @@ private static DataFile buildDataFile(
         .build();
   }
 
-  private static ExecutorService migrationService(int concurrentDeletes) {
+  private static ExecutorService migrationService(int numThreads) {
```

Comment:
Nit: This seems like an unnecessary name change to me, there are other places in the code where concurrentX refers to parallelism we want, and I don't see a need to change that convention

Reply:
However, this service is not for deletes. It might have been copied from somewhere.

Reply:
Ah you are right. I confused myself, in that case this name is a good thing. Thanks for fixing this!